joomla/joomla-cms
# [SEO] Correct heading tag for titles in article view + code styles

**Sets the correct heading tag for the title in the article view if no page heading is provided + code styles**

At the moment the h1 tag is used for the page heading and h2 tag for the title in the article view of the content component. But if no page heading is provided, the title still has only the h2 tag and not the h1 tag as it should be.

I also couldn't resist to clean the file a little bit and improve the code style. So, the main change for the fix is in line 42 and line 46.

**How to test**
1. Create an article
2. Create an link to this article in the menu
3. In the menu entry go to the tab "Page display" and enable the option "Show Page Heading". You should als enter a value in the option "Page Heading" or "Browser Page Title" to see the output in the frontend.
4. Open the article in the frontend with the link from the menu. You will see that the page heading will have an h1 tag and the article title an h2 tag.
5. Now go to the menu entry in the backend and disable the option "Show Page Heading".
6. Reload the article in the frontend and you will see that the page heading is not displayed any more but the article title still has the h2 tag instead of the correct h1 tag.
7. Apply this PR and check again. The title will have the h1 tag.
